server/data/data-jmap-cassandra/src/main/java/org/apache/james/jmap/cassandra/filtering/RuleDTO.java

```diff
@@ -237,7 +239,7 @@ public ActionDTO(@JsonProperty("appendIn") AppendInMailboxesDTO appendIn,
                          @JsonProperty("seen") boolean seen,
                          @JsonProperty("important") boolean important,
                          @JsonProperty("reject") boolean reject,
-                         @JsonProperty("keywords") List<String> keyworkds,
+                         @JsonProperty("keywords") @JsonSetter(nulls = Nulls.AS_EMPTY)  List<String> keyworkds,
```

Comment:
```suggestion
                         @JsonProperty("keywords") @JsonSetter(nulls = Nulls.AS_EMPTY)  List<String> keywords,
```

Well, there is a concern that persisted data using `keyworkds` would be lost. Should be acceptable IMO because this keyword introduction is recent, with no FE implementation yet so no real usage yet.

Reply:
Honestly I spotted it and... I'm not so sure... We have potentially real usage on staging and prod for example. This action has been introduced in a previous release already, we were saving them, just not deserializing them correctly

Reply:
Likely not prod but staging at least... Maybe we could think of fixing it bt being retrocompatible with keyworkds on at least read to not break anything at least. 

But that should be treated in an other ticket/PR then